### PR TITLE
Add click-log

### LINKED ISSUE #13: Proper logging
This issue was moved from the issue tracker at GitLab.

---

Todoman currently has no way of emitting warnings, and a possibility to turn those warnings on and off. Warnings will probably be required for reporting invalid items (the current behavior is basically to ignore those, because logging handlers are not set up properly)

I'd like to rip out some code from vdirsyncer and https://github.com/mitsuhiko/click/pull/278, and make a third-party package on click-contrib to provide such things. I think I'll have to wait for Click 5.0 before implementing this though.

### PR DESCRIPTION
**Pull request by untitaker**
Fix #13

### CODE REVIEW

File: todoman/cli.py [hunk 1]
```diff
@@ -13,6 +15,8 @@
 TODO_ID_MIN = 1
 with_id_arg = click.argument('id', type=click.IntRange(min=TODO_ID_MIN))
 
+logger = logging.getLogger(__name__)
```
Comment (hobarrera) [inline todoman/cli.py]: Do we need this `logger`?

File: todoman/cli.py [hunk 2]
```diff
@@ -70,6 +74,8 @@ def command_wrap(*a, **kw):
 
 
 @click.group(invoke_without_command=True)
+@click_log.init('vdirsyncer')
```
Comment (hobarrera) [inline todoman/cli.py]: `todoman` 😛

**Comment by untitaker** [comment]
Fixed both of your comments.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -7,6 +7,7 @@ releases, in reverse chronological order.
 v2.1.0
 ------
 
+* New PyPI dependency ``click-log``.
```
Comment (hobarrera) [inline CHANGELOG.rst]: Please also mention the new `--verbosity` flag.

File: CHANGELOG.rst [hunk 4]
```diff
@@ -7,6 +7,8 @@ releases, in reverse chronological order.
 v2.1.0
 ------
 
+* The global ``--verbosity`` option has been introduced. It doesn't do much for
```
Comment (hobarrera) [inline CHANGELOG.rst]: :+1: 😆

**Pull request merged**